packages/ember-glimmer/lib/component-managers/template-only.ts

```diff
@@ -0,0 +1,44 @@
+import { VersionedPathReference } from '@glimmer/reference';
+import { CompiledDynamicProgram, ComponentDefinition, NULL_REFERENCE } from '@glimmer/runtime';
+import { Opaque } from '@glimmer/util';
+import Environment from '../environment';
+import { OwnedTemplate, WrappedTemplateFactory } from '../template';
+import AbstractManager from './abstract';
+
+class TemplateOnlyComponentLayoutCompiler {
+  static id = 'template-only';
+
+  constructor(public template: WrappedTemplateFactory) {
+  }
+
+  compile(builder: any) {
+    // TODO: use fromLayout
```

Comment:
I think @krisselden told me `fromLayout` is broken on this version of Glimmer, so I didn't bother.